.github/workflows/task-test.yml

```diff
@@ -74,46 +72,18 @@ jobs:
             fi
           done
           echo "supported=true" >> $GITHUB_OUTPUT
-      - name: Deps checkout (node20)
+      - name: Full checkout (node20)
         if: steps.node20.outputs.supported == 'true'
         uses: actions/checkout@v4
         with:
-          path: setup
-          sparse-checkout-cone-mode: false
-          sparse-checkout: |
-            .install
-            tests/pytests/requirements.*
-      - name: Deps checkout (node20 not supported)
+          submodules: recursive
+      - name: Full checkout (node20 not supported)
         if: steps.node20.outputs.supported == 'false'
         run: |
           # Execute the logic based on the detected platform
           echo "Detected platform: ${{ inputs.container }}"
           case "${{ inputs.container }}" in
-            ubuntu:bionic | amazonlinux:2 | centos:7)
-              echo "Installing prerequisites for ${{ inputs.container }}..."
-
-              # Install git (platform-specific package management)
-              case "${{ inputs.container }}" in
-                ubuntu:bionic)
-                  apt update
-                  apt install -y git
-                  ;;
-                amazonlinux:2)
-                  yum update -y
-                  yum install -y git
-                  ;;
-                centos:7)
-                  # http://mirror.centos.org/centos/7/ is deprecated, so we have to disable mirrorlists
-                  # and change the baseurl in the repo file to the working mirror (from mirror.centos.org to vault.centos.org)
-                  for file in /etc/yum.repos.d/*.repo; do
-                    sed -i 's/^mirrorlist=/#mirrorlist=/g' $file
-                    sed -i 's/^#[[:space:]]*baseurl=http:\/\/mirror/baseurl=http:\/\/vault/g' $file
```

Comment:
Sure this can be moved to `build_artifacts`? What about when we want to test on `centos:7` manually/merge-queue?

Reply:
shouldn't move... it is missing here. Good catch